tests/network/expose.go

```diff
@@ -825,3 +796,22 @@ func generateHelloWorldServer(vmi *v1.VirtualMachineInstance, testPort int, prot
 		&expect.BExp{R: console.RetValue("0")},
 	}, 60)).To(Succeed())
 }
+
+func createAndWaitForJobToSucceed(jobFactory func(host, port string) *batchv1.Job, namespace, ip, port, viaMessage string) error {
+	By(fmt.Sprintf("Starting a job which tries to reach the VMI via the %s", viaMessage))
+	virtClient := kubevirt.Client()
+	jobInstance := jobFactory(ip, port)
+	jobInstance, err := virtClient.BatchV1().Jobs(namespace).Create(context.Background(), jobInstance, metav1.CreateOptions{})
```

Comment:
super nit:

you can drop the client variable:
```suggestion
	jobInstance := jobFactory(ip, port)
	jobInstance, err := kubevirt.Client().BatchV1().Jobs(namespace).Create(context.Background(), jobInstance, metav1.CreateOptions{})
```

Reply:
There are many places like this throughout this file. Maybe it deserves a separate commit

Reply:
The priority is to satisfy the reviewer and in this case you intentionally added a variable which is not a must in this specific function.

Reply:
ah I think I misread what @nunnatsa was asking. 
I was wondering wehther to remove all the unneeded uses of virtClient param. 
Anyways did as you asked.